internal/le/certs.go

```diff
@@ -54,3 +61,19 @@ func GenerateLECerts(serverName, relPath string) error {
 	}
 	return nil
 }
+
+func ReadCerts(certFolder string) (*CertList, error) {
```

Comment:
match method name with GenerateLECerts 
or rename GenerateLECerts to GenerateCerts